games-util/lutris/lutris-9999.ebuild

```diff
@@ -49,14 +52,10 @@ src_install() {
 	distutils-r1_src_install
 }
 
-pkg_preinst() {
-	gnome2_icon_savelist
-	gnome2_schemas_savelist
-}
-
 pkg_postinst() {
 	gnome2_icon_cache_update
 	gnome2_schemas_update
+	xdg_desktop_database_update
 
 	elog "For a list of optional dependencies (runners) see:"
```

Comment:
I could not find any "optional dep" in README.rst so I changed the text in my version:

```
# Quote README.rst
elog "Lutris installations are fully automated through scripts, which can"
elog "be written in either JSON or YAML. The scripting syntax is described"
elog "in ${EROOT%/}/usr/share/doc/${PF}/installers.rst.bz2, and is also"
elog "available online at lutris.net."
```